packages/message-system/src/scripts/sign-config.ts

```diff
@@ -13,7 +13,7 @@ let JWS_PRIVATE_KEY: string;
 if (isCodesignBuild) {
     console.log('Signing config using production private key!');
 
-    JWS_PRIVATE_KEY = fs.readFileSync(process.env.JWS_PRIVATE_KEY_FILE!, 'utf-8');
+    JWS_PRIVATE_KEY = process.env.JWS_PRIVATE_KEY_ENV!;
```

Comment:
This will stop working in GitLab, unless you've added the JWS_PRIVATE_KEY_ENV there as well, right? So we should probably delete all the `msg-system ...` jobs from GitLab CI in this PR too?

Reply:
Or do we want to test this first and delete it from GitLab later?

Reply:
I would test it first and then delete it if its working.

Reply:
Will this even work btw? Im not completeley sure if its ok to provide string from env there like this.

Reply:
yeah, it should work